nmigen/hdl/ir.py

```diff
@@ -532,12 +532,12 @@ def prepare(self, ports=None, missing_domain=lambda name: ClockDomain(name)):
         if ports is None:
             fragment._propagate_ports(ports=(), all_undef_as_ports=True)
         else:
-            # typecheck ports is either list or tuple (#362)
+            # typecheck ports is either list or tuple
```

Comment:
Now this comment is just tautological with the code that follows it :)

Reply:
:P